docs/project/library-servicing.md

```diff
@@ -41,5 +41,14 @@ All the servicing change must go through an approval process. You have two ways
 For both cases, you must:
 
 - Fill out the template of the PR description.
-- Add the `servicing-consider` label.
-- Bring it to the attention of the engineering lead responsible for the area, so they consider the fix for servicing.
\ No newline at end of file
+- Bring it to the attention of the [engineering lead responsible for the area](~/docs/area-owners.md).
+- If the fix is a product change, the area owner will:
+  - Add the `Servicing-consider` label.
+  - Request approval to merge this servicing change.
```

Comment:
These are a little vague. You can say "request tactics approval", and "If the change is approved, a tactics representative will replace..."

Reply:
I thought about this. The document is meant for external folks, who have no knowledge of Tactics. But I can mention it under the assumption that we don't need to give a longer explanation of what Tactics is, but the area owner would know.

Reply:
Sorry I deleted the other comment, I noticed I placed it in the wrong location. Adding the suggestion here. @hoyosjs what do you think?

```suggestion
  - Ask the area owner to champion your PR in the .NET Tactics meeting to request merge approval.
```